djangocms_audio/models.py

```diff
@@ -0,0 +1,170 @@
+# -*- coding: utf-8 -*-
+from django.db import models
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from django.utils.encoding import python_2_unicode_compatible
+from django.utils.translation import ugettext, ugettext_lazy as _
+
+from cms.models import CMSPlugin
+
+from djangocms_attributes_field.fields import AttributesField
+
+from filer.fields.file import FilerFileField
+from filer.fields.folder import FilerFolderField
+
+
+"""
+Enables the user to add an "Audio player" plugin that serves as
+a wrapper rendering the player and its options.
+
+The "Audio player" plugin allows to add either a single "File" or a reference
+to a "Folder" as children.
+"""
+
+
+# mp3 is supported by all major browsers
+ALLOWED_EXTENSIONS = getattr(
+    settings,
+    'DJANGOCMS_AUDIO_ALLOWED_EXTENSIONS',
+    ['mp3', 'ogg'],
+)
+
+
+# Add additional choices through the ``settings.py``.
+def get_templates():
+    choices = getattr(
+        settings,
+        'DJANGOCMS_AUDIO_TEMPLATES',
+        False,
```

Comment:
set this to a `[]` instead of `False`.
Then you can avoid the extra conditional and just `return choices`